plugins/argentina-afip-invoices/src/manifest.json

```diff
@@ -5,13 +5,13 @@
         "displayName": "Facturas Argentina AFIP",
         "description": "Plugin para obtener CAE (Facturacion electronica) en Argentina",
         "url": "https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/argentina-afip-invoices",
-        "version": "1.0.1",
+        "version": "1.0.2",
         "ucrmVersionCompliancy": {
             "min": "2.15.0-beta3",
             "max": null
         },
       "unmsVersionCompliancy": {
```

Comment:
Hello @fgampel, thank you for the PR. To make the plugin incompatible with UNMS, you can just remove the `unmsVersionCompliancy` from manifest altogether. Take a look here as an example https://github.com/Ubiquiti-App/UCRM-plugins/blob/master/plugins/routeros-packet-manager/src/manifest.json#L9

Reply:
Thanks @keksa , in the past i always let this file without UNMS compatibility, but i don't know why, someone added this compatibility, and after that, i had many many emails in my inbox telling me that the plugin is not working haha!

Updated again, now i think plugin.json is in right codding.

Thanks